CMSgov/mdct-mcr
# Slight edits for how reports are fetched in AddEditModal

## Description
Missed Brax's comments [here](https://github.com/CMSgov/mdct-mcr/pull/3854) as I merged a moment before he put in his review. This PR addresses his feedback

### How to test
./dev local
Login as a state user
Go to the dashboard
Create 2 programs with whatever info you like
Click the edit button next to them to see the fields pre populated
Click the add program button to make sure its empty if not editing
Enter each program and make sure they are indeed separate.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
